internal/core/src/exec/operator/groupby/SearchGroupByOperator.h

```diff
@@ -140,7 +140,7 @@ PrepareVectorIteratorsFromIndex(const SearchInfo& search_info,
                     index.VectorIterators(dataset, search_conf, bitset);
             if (iterators_val.has_value()) {
                 search_result.AssembleChunkVectorIterators(
-                    nq, 1, -1, iterators_val.value());
+                    nq, 1, {0}, iterators_val.value());
```

Comment:
This change results in different behavior. Change this to `{}` or check `if (total_rows_until_chunk_.size() == 1` in the function `convert_to_segment_offset()`

Reply:
> This change results in different behavior. Change this to `{}` or check `if (total_rows_until_chunk_.size() == 1` in the function `convert_to_segment_offset()`

ok, this does not affect correctness and I will change it in next pr.